flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -358,8 +370,9 @@ public void testBatchQueryExecution() throws Exception {
         }
     }
 
-    @Test(timeout = 90_000L)
-    public void testBatchQueryExecutionMultipleTimes() throws Exception {
+    @Test
+    @Timeout(90)
```

Comment:
Since JUnit 5 supports specification of timeunits it would be much more convenient (code will be more readable) to use them like 
```java
@Timeout(value = 90, unit = TimeUnit.SECONDS)
```
or
```java
@Timeout(value = 90, unit = TimeUnit.MILLISECONDS)
```

Reply:
As @XComp suggested:
> Keep in mind that we actually want to remove timeouts from JUnit tests (see [coding guidelines](https://flink.apache.org/contributing/code-style-and-quality-common.html#avoid-timeouts-in-junit-tests)) to benefit from the thread dump printout generated by our CI tools. So it might be better to remove them instead of migrating them to JUnit5

We'd better remove timeouts instead of migrating them to JUnit5.